rainier-core/src/main/scala/com/stripe/rainier/compute/Compiler.scala

```diff
@@ -24,34 +24,6 @@ trait Compiler {
                     outputs: Seq[Real]): ir.CompiledFunction
 }
 
-final case class InstrumentingCompiler(orig: Compiler, printEvery: Int)
```

Comment:
is this an incidental change to this commit?

Reply:
Sorta. I would have had to change this class and I don't use it anymore so it was faster to delete it.